libxfdashboard/live-workspace.c

```diff
@@ -345,8 +345,11 @@ static void _xfdashboard_live_workspace_on_clicked(XfdashboardLiveWorkspace *sel
 
 	action=XFDASHBOARD_CLICK_ACTION(inUserData);
 
-	/* Only emit signal if click was perform with left button */
-	if(xfdashboard_click_action_get_button(action)==XFDASHBOARD_CLICK_ACTION_LEFT_BUTTON)
+	/* Only emit any of these signals if click was perform with left button 
+	 * or is a short touchscreen touch event.
+	 */
+	if(xfdashboard_click_action_get_button(action)!=XFDASHBOARD_CLICK_ACTION_LEFT_BUTTON &&
```

Comment:
This is not correct replaced. The original code checks for left-button but the replaced code checks for non-left-button.

It should be:

```
if(xfdashboard_click_action_get_button(action)==XFDASHBOARD_CLICK_ACTION_LEFT_BUTTON ||
    xfdashboard_click_action_get_button(action)==0)
```